File: classes/parte-7-paradigmas/117-declarativo-consultas-y-transformacion-sql/implementaciones/c/main.c
#include <stdio.h>

int main(void) {
    long suma = 0, x;
    while (scanf("%ld", &x) == 1) {
        if (x % 2 == 0) suma += x;
    }
    printf("suma_pares=%ld\n", suma);
    return 0;
}

File: classes/parte-7-paradigmas/117-declarativo-consultas-y-transformacion-sql/implementaciones/typescript/main.ts
import { readFileSync } from "node:fs";

const nums: number[] = readFileSync(0, "utf8").trim().split(/\s+/).map(Number);
const suma = nums.filter((x) => x % 2 === 0).reduce((a, b) => a + b, 0);
console.log(`suma_pares=${suma}`);
